pkg/v1/providers/ytt/03_customizations/02_avi/add-avi-labels.yaml

```diff
@@ -0,0 +1,20 @@
+#@ load("@ytt:data", "data")
+#@ load("@ytt:json", "json")
+#@ load("@ytt:overlay", "overlay")
+#@ load("@ytt:template", "template")
+
+
+#! Label workload cluster if AVI_LABELS is specified
+
+#@ if data.values.PROVIDER_TYPE == "vsphere" and data.values.TKG_CLUSTER_ROLE == "workload" and data.values.AVI_LABELS:
+#@overlay/match missing_ok=True,by=overlay.subset({"kind":"Cluster"})
+---
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: Cluster
+metadata:
+  #@overlay/match-child-defaults missing_ok=True
+  labels:
+    _: #@ template.replace(data.values.AVI_LABELS)
```

Comment:
Just want to confirm that this will append values in `AVI_LABELS` not replace it with `AVI_LABELS`, right?
Cuz we shouldn't replace it with `AVI_LABELS`

Reply:
No it won't replace. instead, it will append the AVI_LABELS to the original cluster label.